setup.py

```diff
@@ -40,11 +40,20 @@
         "sentry": "sentry-sdk>=0.14.4",
         "spooky": "spooky>=2.0.0",
         "test": [
+            "nose>=1.3.7",
             "sentry-sdk>=0.14.4",
             "PyHamcrest",
             "coverage",
-            "parameterized"
+            "parameterized",
+            "microcosm-metrics>=2.5.1"
```

Comment:
See test setup in microcosm_flask/tests/test_metrics.py
If we do not import microcosm-metrics - metrics tests are skipped.

Reply:
There were no changes to the tests?

Reply:
Not sure I undrstand the question. 
I can pip install `microcosm-metrics` in the `test` section in `entrypoint.sh` if that makes more sense

Reply:
There are changes to the tests

Reply:
Ah, never mind.